src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -42,6 +44,35 @@ class Lambda {
    * @return whether a piece of content was purged or not
    */
   private def sendPurgeRequest(contentId: String): Boolean = {
+    val request = basicPurgeRequest(contentId).build()
+
+    val response = httpClient.newCall(request).execute()
+    println(s"Sent purge request for content with ID [$contentId]. Response from Fastly API: [${response.code}] [${response.body.string}]")
+
+    val purged = response.code == 200
+    purged
+  }
+
+  /**
+   * Send a soft purge request to Fastly API.
+   *
+   * See [[https://docs.fastly.com/guides/purging/soft-purges Fastly soft purging]].
+   *
+   * @return whether a piece of content was purged or not
+   */
+  private def sendSoftPurgeRequest(contentId: String): Boolean = {
+    val request = basicPurgeRequest(contentId)
```

Comment:
it is, see https://github.com/guardian/fastly-cache-purger/pull/3/files/7b3e0bf8a016ae25325444b15a2c1decb6187bdf#diff-dec464a104a3e57383b9d6e6dd67cc52R82